test/test.config.js

```diff
@@ -12,7 +12,7 @@ config.karma.suites['bedrock-vue-credential-creator-wizard'] =
 // TODO: consider moving to `bedrock-karma`?
 config.karma.config.webpack.resolve = {
   alias: {
-    'quasar-framework$': path.resolve(
-      __dirname, 'node_modules/quasar-framework/dist/umd/quasar.mat.umd.js')
+    quasar$: path.resolve(
+      __dirname, 'node_modules/quasar/dist/umd/quasar.mat.umd.js')
```

Comment:
this file does not exist:
node_modules/quasar/dist/umd/quasar.mat.umd.js

 ls node_modules/quasar/dist/
api				quasar.addon.rtl.min.css	quasar.rtl.css
babel-transforms		quasar.common.js		quasar.rtl.min.css
icon-set			quasar.css			quasar.styl
lang				quasar.esm.js			quasar.umd.js
quasar.addon.css		quasar.ie.polyfills.js		quasar.umd.min.js
quasar.addon.min.css		quasar.ie.polyfills.umd.min.js	vetur
quasar.addon.rtl.css		quasar.min.css

Reply:
ok file name is now quasar.umd.js